Mage/src/main/java/mage/abilities/common/DiesAttachedTriggeredAbility.java

```diff
@@ -21,6 +21,7 @@ public class DiesAttachedTriggeredAbility extends TriggeredAbilityImpl {
     private final String attachedDescription;
     private final boolean diesRuleText;
     protected SetTargetPointer setTargetPointer;
+    private final boolean rememberSource; // if true, setTargetPointer will be set to the aura/equipement.
```

Comment:
This seems like the sort of thing that ideally ought to be handled by a `SetTargetPointer` case?  Unfortunately that enum has no comments. Might be more trouble than it's worth

Reply:
I could introduce another SetTargetPointer enum field ATTACHMENT i guess?
PERMANENT_TARGET and ATTACHED_TO_CONTROLLER have only 2 usage each.